bundles/org.openhab.binding.matter/README.md

```diff
@@ -0,0 +1,472 @@
+net.ipv6.conf.wlan0.accept_ra_rt_info_max_plen = 64
+```
+
+To make these changes permanent, add the following lines to `/etc/sysctl.conf`:
+
+```
+net.ipv6.conf.eth0.accept_ra=1
+net.ipv6.conf.eth0.accept_ra_rt_info_max_plen=64
+```
+
+Raspberry Pi users may need to add the following lines to `/etc/dhcpcd.conf` to prevent dhcpcd from overriding the accept_ra value:
+
+```
+noipv6
+noipv6rs
+```
+
+***NOTE:  Please ensure you use the right interface name for your network interface.*** The above examples use `wlan0` and `eth0` as examples.  You can find the correct interface name by running `ip a` and looking for the interface that has an IPv6 address assigned to it. 
+
+## Binding Configuration
+
+This binding does not require any general configuration settings.
+
+## Thing Configuration
+
+### Controller Thing Configuration
+
+The controller thing must be created manually before devices can be discovered.
+
+| Name                      | Type    | Description                                                                                                                       | Default | Required | Advanced |
+|---------------------------|---------|-----------------------------------------------------------------------------------------------------------------------------------|---------|----------|----------|
+| nodeId                    | number  | The matter node ID for this controller                                                                                            | 0       | yes      | no       |
+
+### Node Thing Configuration
+
+Endpoints are discovered automatically (see [Discovery](#Discovery) for more information) and should not be added manually.
+
+| Name       | Type   | Description                        | Default | Required | Advanced |
+|------------|--------|------------------------------------|---------|----------|----------|
+| nodeId     | text   | The node ID of the endpoint        | N/A     | yes      | no       |
+
+### Endpoint Thing Configuration
+
+ Endpoints are discovered automatically once their parent Node has been added (see [Discovery](#Discovery) for more information) and should not be added manually.
+
+| Name       | Type   | Description                        | Default | Required | Advanced |
+|------------|--------|------------------------------------|---------|----------|----------|
+| endpointId | number | The endpoint ID within the node    | N/A     | yes      | no       |
+
+Note: The `bridged-endpoint` thing type is deprecated in favor of the `endpoint` thing type and will be removed in a future release.
+
+## Thing Actions
+
+### Node Thing Actions
+
+| Name        Description                         |                                                                                                                                                                                                                                                                           |
```

Comment:
There is something wrong with the formatting here, the two headers are combined in one.